doc/mkdocs/docs/api/macros/json_legacy_comparison.md

```diff
@@ -0,0 +1,51 @@
+# JSON_LEGACY_COMPARISON
+
+```cpp
+#define JSON_LEGACY_COMPARISON /* value */
+```
+
+This macro enables the (incorrect) legacy comparison behavior of discarded JSON values.
+Possible values are `1` to enable or `0` to disable (default).
+
+When enabled, comparisons involving at least one discarded JSON value yield results as follows:
+
+| **Operator** | **Result**    |
+|--------------|---------------|
+| `==`         | `#!cpp false` |
+| `!=`         | `#!cpp true`  |
+| `<`          | `#!cpp false` |
+| `<=`         | `#!cpp true`  |
+| `>=`         | `#!cpp true`  |
+| `>`          | `#!cpp false` |
+
+Otherwise, comparisons involving involving at least one discarded JSON value always yield `#!cpp false`.
+
+## Default definition
+
+The default value is `0`.
+
+```cpp
+#define JSON_LEGACY_COMPARISON 0
+```
+
+When the macro is not defined, the library will define it to its default value.
+
+## Notes
+
+!!! warning "Inconsistent behavior in C++20-mode"
+
+    The `<=>` operator will always yield the correct result (`#!cpp false`) but overloads
```

Comment:
Does `<=>` ignore `JSON_LEGACY_COMPARISON`?

Reply:
Yes. I did not want to hack the legacy behavior into the new, clean comparison function. It would also make an already slight more expensive operation even more expensive, since I'd have to do some upfront checks first, then do the already more expensive 3-way comparison.